rfcs/013-gitlab-spam-checker/013-gitlab-spam-checker.md

```diff
@@ -0,0 +1,93 @@
+# Implementing a Spam Management Service for GitLab
+
+- Start Date: 2023-05-19
+- Status: Proposed
+
+## Summary
+
+This proposal aims to introduce a spam management service to mitigate spam on GitLab and improve user experience and security. This service will use machine learning models to detect and eliminate spam across the platform in real-time, focusing on snippets, issues, notes, projects, and user profiles.
```

Comment:
I think it would make sense to quality real-time here; further down, you mention a mix of hook based mechanisms plus mechanisms which are schduled, so I guess it's not all real-time; also I guess some wil require human intervention in which case the detection cuold be somehow real-time but the elimination might not be.

Reply:
The spam management service shouldn't require human intervention. Where possible, hooks are used so its real time, and where GitLab doesn't supply hooks, such as for snippets, we'll use jobs at regular interviews